### PR TITLE
Add cosmos DB sample

### PR DESCRIPTION
**Pull request by mcardosos**
Thanks for your contribution! Please let us know what problems this PR addresses or what functionality it adds. Please link to other related issues and PRs to speed up review. Thank you!

### CODE REVIEW

File: cosmos/cosmos_test.go [hunk 1]
```diff
@@ -4,3 +4,129 @@
...
+	}
+	os.Exit(m.Run())
+}
+
+func ExampleCosmosDBOperations() {
+	accountName := strings.ToLower(accountName)
+	helpers.SetResourceGroupName("CosmosDBOperations")
+	ctx := context.Background()
+	defer resources.Cleanup(ctx)
+	_, err := resources.CreateGroup(ctx, helpers.ResourceGroupName())
```
Comment (jhendrixMSFT) [inline cosmos/cosmos_test.go]: I noticed that in most cases the result returned from the API isn't captured and I'm curious as to why.

Comment (mcardosos) [reply cosmos/cosmos_test.go]: Well, it is not really needed (I have nothing to do with the `ResourceGroup` struct here)

File: cosmos/mongo.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+package cosmos
+
+import (
+	"github.com/globalsign/mgo"
+	"github.com/globalsign/mgo/bson"
+)
+
+// GetCollection gets a mongoDB collection
+func GetCollection(session *mgo.Session, database, collectionName string) (*mgo.Collection, error) {
```
Comment (jhendrixMSFT) [inline cosmos/mongo.go]: Seems like the error return value is unnecessary.

Comment (mcardosos) [reply cosmos/mongo.go]: D: Fixed

**Pull request merged**